lib/keila/mailings/sender_adapters/schema.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Keila.Mailings.SenderAdapters.Schema do
```

Comment:
I’m not quite sure if this module is actually necessary. It seems like it is `use`d only once, so you might as well include `sender_adapters_fields` directly in `Keila.Mailings.Sender.Config` to avoid unnecessary complexity.

This should work directly in the schema definition:

```elixir
SenderAdapters.schema_fields() |> Enum.each(fn {name, type} -> field(name, type) end)
```
```